src/routers/features/cgn/index.ts

```diff
@@ -91,6 +98,22 @@ addHandler(cgnRouter, "get", addPrefix("/activation"), (_, res) =>
         instance_id: { id },
         status: StatusEnum.COMPLETED
       };
+
+      const currentPreference = servicesPreferences.get(
+        cgnServiceId as ServiceId
+      );
+
+      const increasedSettingsVersion = currentPreference
+        ? (((currentPreference.settings_version as number) +
+            1) as ServicePreference["settings_version"])
+        : (0 as ServicePreference["settings_version"]);
```

Comment:
```suggestion
      const increasedSettingsVersion = ((currentPreference?.settings_version ??
        -1) + 1) as NonNegativeInteger;
```
the same, shorter but less readable. On your flavor